test/unit/ReservoirRouter.t.sol

```diff
@@ -0,0 +1,759 @@
+pragma solidity 0.8.13;
+
+import "v3-core/test/__fixtures/BaseTest.sol";
+
+import { WETH } from "solmate/tokens/WETH.sol";
+import { FixedPointMathLib } from "solmate/utils/FixedPointMathLib.sol";
+import { Math } from "@openzeppelin/utils/math/Math.sol";
+
+import { MathUtils } from "v3-core/src/libraries/MathUtils.sol";
+
+import { IReservoirPair } from "v3-core/src/interfaces/IReservoirPair.sol";
+import { ExtraData } from "src/interfaces/IReservoirRouter.sol";
+import { ReservoirLibrary, IGenericFactory } from "src/libraries/ReservoirLibrary.sol";
+import { ReservoirRouter } from "src/ReservoirRouter.sol";
+
+contract ReservoirRouterTest is BaseTest
+{
+    WETH            private _weth   = new WETH();
+    ReservoirRouter private _router = new ReservoirRouter(address(_factory), address(_weth));
+
+    bytes[]         private _data;
+
+    // required to receive ETH refunds from the router
+    receive() external payable {}
+
+    function testAddLiquidity(uint256 aTokenAMintAmt, uint256 aTokenBMintAmt) public
+    {
+        // assume
+        uint256 lTokenAMintAmt = bound(aTokenAMintAmt, 1, type(uint112).max);
+        uint256 lTokenBMintAmt = bound(aTokenBMintAmt, 1, type(uint112).max);
+
+        // arrange
+        _tokenA.mint(_bob, lTokenAMintAmt);
+        _tokenB.mint(_bob, lTokenBMintAmt);
+
+        vm.startPrank(_bob);
+        _tokenA.approve(address(_router), type(uint256).max);
+        _tokenB.approve(address(_router), type(uint256).max);
+
+        // act
+        _data.push(abi.encodeCall(
+            _router.addLiquidity,
+            (
+                address(_tokenA),
+                address(_tokenB),
+                0,
+                lTokenAMintAmt,
+                lTokenBMintAmt,
+                1,
+                1,
+                _bob
+            )
+        ));
+
+        bytes[] memory lResult = _router.multicall(_data);
+
+        // assert
+        ReservoirPair lPair = ReservoirPair(
+                                ReservoirLibrary.pairFor(
+                                    address(_factory),
+                                    address(_tokenA),
+                                    address(_tokenB),
+                                    0
+                                ));
```

Comment:
I would expect justindent by 4, rather than align hanging

Reply:
so, like that?

```
ReservoirPair lPair = ReservoirPair(
    ReservoirLibrary.pairFor(
        address(_factory),
        address(_tokenA),
        address(_tokenB),
        0
));
```